client/js/admin.ts

```diff
@@ -642,6 +645,61 @@ for (let i = 0; i < timeInputs.length; i++) {
 	timeInputs[i].value = moment(new Date(timeInputs[i].dataset.rawValue || "")).format("Y-MM-DDTHH:mm:00");
 }
 
+// Uncheck available confirmation branches for application branch when "skip confirmation" option is selected
+function uncheckConfirmationBranches(applicationBranch: string) {
+	let checkboxes = document.querySelectorAll(`.branch-role[data-name="${applicationBranch}"] .availableConfirmationBranches input[type="checkbox"]`);
```

Comment:
Types will work better here if it's `let checkboxes: NodeListOf<HTMLInputElement> = x`. Then you don't need the ugly cast on line 652.

Reply:
Typescript won't allow me to cast `NodeListOf<Element>` to `NodeListOf<HTMLInputElement>`